### DISCUSSION TITLE
Differences to vue-chartjs v4

### DISCUSSION THREAD

**Discussion by dargmuesli** [victorgarciaesgi/vue-chart-3#117] [Q&A] [answered]
Hey there,
now that https://github.com/apertureless/vue-chartjs v4 includes chartjs [v3 support](https://github.com/apertureless/vue-chartjs/issues/695#issuecomment-1078969043) I'm wondering what there is that makes this fork different to the original project.
I had switched to your project @victorgarciaesgi and am very thankful for your work to provide chartjs v3 support so quickly. I also like deduplication so that's where my question arises.
Is there still an advantage in using your fork?
Thank you once more,
greetings from Germany!

**Comment by victorgarciaesgi** [comment, marked as answer]
Thanks a lot for your words! Indeed `vue-chartjs` updated to v3 support and the api it's pretty much the same as me. I don't have the same free time to invest into this lib so I advise you to switch to `vue-chartjs` as they are certainly more active with a bigger userbase so for the long run it's better for you :)

**Reply by dargmuesli** [reply]
For people from the future: one difference would currently be that `vue-chartjs` does not yet support reactive props: https://github.com/apertureless/vue-chartjs/issues/805
